modules/CmpdReg/src/client/custom/Lot_Custom.js

```diff
@@ -524,6 +522,8 @@ $(function () {
         if (this.vendorCodeController.getSelectedModel().isNew()) vendor = null;
         else vendor = this.vendorCodeController.getSelectedModel();
         this.model.set({
+          notebookPage: jQuery.trim(this.$('.notebookPage').val()),
+          synthesisDate: jQuery.trim(this.$('.synthesisDate').val()),
```

Comment:
Chemist was set below already so just needed to add these lines to get the notebook page and synthesis date. always.